side_runner_py/hook.py

```diff
@@ -45,38 +45,41 @@ def _run_hook(pre_or_post, kind, match_funcs, current_test_dict):
                 call_with_argname_bind(func, current_test_dict)
 
 
-def run_hook_per_project(pre_or_post, test_project):
+def run_hook_per_project(pre_or_post, session_manager, test_project):
```

Comment:
@rsp9u @yaamai May be session_manager can me kept as last argument and make it optional?

Reply:
So that you don't have to modify all UTs

Reply:
The instance of SessionManager lives while side-runner-py running. This argument is always passed, so I think it should be positional arugment.